docs/api.rst

```diff
@@ -12,7 +12,6 @@ Consult the `notebooks <https://github.com/CQCL/pytket-quantinuum/tree/develop/e
 
 .. autoclass:: QuantinuumBackend
     :show-inheritance:
-    :special-members: __init__
```

Comment:
This removes the documentation for the arguments. Is this intended?

Reply:
This is intended. the args for initialising `QuantinuumBackend` are documented twice.
https://tket.quantinuum.com/extensions/pytket-quantinuum/api.html#pytket.extensions.quantinuum.QuantinuumBackend

Reply:
Changed it back to show the `__init__` method